src/Symfony/Installer/NewCommand.php

```diff
@@ -412,4 +414,19 @@ protected function getRemoteFileUrl()
     {
         return 'http://symfony.com/download?v=Symfony_Standard_Vendors_'.$this->version;
     }
+
+    /**
+     * Updates the 'hash' value stored in composer.lock to avoid out-of-sync
+     * problems when the composer.json file contents are changed.
+     */
+    private function syncComposerLockFile()
+    {
+        $composerFileContents = file_get_contents($this->projectDir.'/composer.json');
+        $lockFileContents = json_decode(file_get_contents($this->projectDir.'/composer.lock'), true);
+
+        $hash = md5($composerFileContents);
```

Comment:
I found it here: https://github.com/composer/composer/blob/69210d5bc130f8cc9f96f99582a041254d7b9833/src/Composer/Factory.php#L309  But to be sure I made an `md5` of several `composer.json` files and they matched the values stored in `composer.lock`.